### ISSUE TITLE
Task Repository Schema not uptodate 

### ISSUE THREAD

**Issue by imod** [spring-cloud/spring-cloud-task#627]
The ERD shown at https://docs.spring.io/spring-cloud-task/docs/2.1.1.RELEASE/reference/htmlsingle/#appendix-task-repository-schema is not uptodate. The following columns are missing:

- `TASK_EXECUTION.external_execution_id`
- `TASK_EXECUTION.parent_execution_id`

Also, the following things are not documented:

- indices
- initial values (e.g. in `TASK_SEQ`)

It would be very helpful if you could document the whole DDL and not just show the ERD. The reasons are the same as I reported for SCDF at https://github.com/spring-cloud/spring-cloud-dataflow/issues/3460

In short there always are situations where one needs to create the DB objects by hand, in our case we need to create each table with the correct `ROW_FORMAT` defined. If we don't do so, the creation of indices fails with `Index column size too large. The maximum column size is 767 bytes.`. The whole problem is also documented here: https://community.pivotal.io/s/article/Apps-are-down-due-to-the-Maximum-Column-Size-is-767-bytes-Constraint-in-MySQL

For sure there are other situations where it is required to create the tables in some specific way.

**Comment by sabbyanandan** [comment]
Thank you for the feedback! We will get it updated soon.

**Comment by sabbyanandan** [comment]
Addressed in https://github.com/spring-cloud/spring-cloud-task/commit/d1827eab258a3c89a4199f10f003059bb8765fe3.
